sdk/dapp-kit/src/components/icons/SuiIcon.tsx

```diff
@@ -5,12 +5,12 @@ import type { ComponentProps } from 'react';
 
 export function SuiIcon(props: ComponentProps<'svg'>) {
 	return (
-		<svg width={28} height={28} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
-			<rect width={28} height={28} rx={6} fill="#6FBCF0" />
+		<svg xmlns="http://www.w3.org/2000/svg" width={32} height={32} fill="none" {...props}>
+			<path fill="#4CA3FF" d="M0 0h32v32H0z" />
```

Comment:
This version of the icon is just a normal rectangle, do we want the rounded corners to be part of the svg, or do want to add the rounding via css or something to the icon after?

Reply:
never mind, just manually edited the svg